test_util/aws.py

```diff
@@ -424,6 +424,7 @@ class VpcCfStack(CfStack):
     @classmethod
     def create(cls, stack_name, instance_type, instance_os, instance_count,
                admin_location, key_pair_name, boto_wrapper):
+
```

Comment:
It'd be helpful to separate style changes from behavior changes in separate commits.